modules/ztp-site-cleanup.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="ztp-site-cleanup_{context}"]
+= Site cleanup
+
+To remove a site and the associated installation and policy custom resources (CRs), remove the SiteConfig and site-specific PolicyGenTemplate CR from the Git repository. The pipeline hooks will remove the generated CRs.
```

Comment:
s/SiteConfig and PolicyGenTemplate CR/`SiteConfig` and `PolicyGenTemplate` CRs

"The pipeline hooks remove the generated CRs."

Reply:
changes made